autode/utils.py

```diff
@@ -399,36 +441,96 @@ def wrapped_function(*args, **kwargs):
     return wrapped_function
 
 
-def timeout(seconds: float, return_value: Optional[Any] = None) -> Any:
+def _cleanup_after_exp_timeout_win() -> None:
     """
-    Function decorator that times-out after a number of seconds
+    If experimental timeout has been used, the ProcessPool has to be shutdown
+    otherwise File permission errors will be caused on Windows
+    """
+    if Config.use_experimental_timeout:
+        pool = loky.get_reusable_executor()
+        pool.shutdown()
```

Comment:
would be nice to add a test here that when a `ProcessPool` is created with a timeout and hits it then these lines are executed ok